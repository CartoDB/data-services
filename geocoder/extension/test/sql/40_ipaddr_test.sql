-- Check that the geocoding function is callable, should return success = false
SELECT (geocode_ip(Array['100.0.24.0'])).*;

-- Add a few IP sources
COPY ip_address_locations (network_start_ip, the_geom, cartodb_id, created_at, updated_at, the_geom_webmercator) FROM stdin;
::ffff:2.235.35.0	0101000020E610000072F90FE9B7CF22405DFE43FAEDC34640	2821226	2014-08-25 10:35:51.665546+00	2014-08-25 10:35:51.665546+00	0101000020110F000010801778FBF32F4109868FF8BCC35541
::ffff:31.7.187.0	\N	2783250	2014-08-25 10:35:51.665546+00	2014-08-25 10:35:51.665546+00	\N
::ffff:64.110.146.0	\N	2783251	2014-08-25 10:35:51.665546+00	2014-08-25 10:35:51.665546+00	\N
::ffff:72.5.198.0	\N	2783252	2014-08-25 10:35:51.665546+00	2014-08-25 10:35:51.665546+00	\N
::ffff:77.73.184.0	\N	2783253	2014-08-25 10:35:51.665546+00	2014-08-25 10:35:51.665546+00	\N
\.

-- Check that the geocoding function is callable, should return success = true
SELECT (geocode_ip(Array['2.235.35.0'])).*;
