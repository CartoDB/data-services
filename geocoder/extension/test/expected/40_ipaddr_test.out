-- Check that the geocoding function is callable, should return success = false
SELECT (geocode_ip(Array['100.0.24.0'])).*;
     q      | geom | success 
------------+------+---------
 100.0.24.0 |      | f
(1 row)

-- Add a few IP sources
COPY ip_address_locations (network_start_ip, the_geom, cartodb_id, created_at, updated_at, the_geom_webmercator) FROM stdin;
-- Check that the geocoding function is callable, should return success = true
SELECT (geocode_ip(Array['2.235.35.0'])).*;
     q      |                        geom                        | success 
------------+----------------------------------------------------+---------
 2.235.35.0 | 0101000020E610000072F90FE9B7CF22405DFE43FAEDC34640 | t
(1 row)

